test/index.js

```diff
@@ -656,6 +656,15 @@ describe('either', function() {
       // jshint +W053
     });
 
+    it('provides an "extend" method', function() {
+      eq(S.Left(42).extend(function(x) { return x / 2; }), S.Left(42));
```

Comment:
Let's begin by asserting arity:

```javascript
eq(S.Left('abc').extend.length, 1);
```

I'd also prefer to use `S.Left('abc')` in the `x / 2` example above, since a Left will typically contain an error message.